vnext/JSI.Desktop.UnitTests/RunUnitTests.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
either add correct copyright in all files, or don't add them anywhere. Copyright usually requires a year, not sure what current Microsoft GitHub licensing requirements are.

Reply:
This is the same Microsoft copyright header that is used in the rest of Microsoft/react-native-windows. @Khalef1 do you know what we need to do here?

Reply:
I don't. But it should be consistent across all files that we own. So if you see a pattern already, go with that. Otherwise, let's take it offline and we can ask for guidance.

Reply:
Again, this is the header is used everywhere else in Microsoft/react-native-windows as well as Microsoft/react-native. Let's take this offline and ask the right people for guidance.

---
In reply to: [298783056](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r298783056) [](ancestors = 298783056)